### DISCUSSION TITLE
Template require is not reloaded in watch mode

### DISCUSSION THREAD

**Discussion by wh81752** [Mogztter/asciidoctor-web-pdf#484] [Ideas]
I use options -w (watch) and --preview as shown below:

```console
$ asciidoctor-web-pdf -w --preview --template-require ./doc.js doc.adoc
Watch mode entered, needs to be manually terminated using Ctrl+C!
```

Now, when I change `doc.adoc` then my changes are reflected in Chromium opened by `--preview`. That is expected.

However, my (structural) changes to `doc.js` are not taken into account. So whenever I change `doc.js` I need to terminate process above and restart again. 

Is that expected behaviour  or do I miss something?

**Comment by ggrossetie** [comment]
I never thought about that... 

> However, my (structural) changes to doc.js are not taken into account. So whenever I change doc.js I need to terminate process above and restart again.

Yes, not ideal...

> Is that expected behaviour or do I miss something?

I can confirm that, currently, we do not reload the template require file but I think it should be possible to do it. 
I've created an issue to track progress: https://github.com/Mogztter/asciidoctor-web-pdf/issues/540
